Dockerfile

```diff
@@ -4,7 +4,7 @@ WORKDIR /code
 COPY poetry.lock pyproject.toml /code/
 RUN pip install poetry
```

Comment:
may be good to add the version of poetry?

Reply:
tbh I prefer if for now we use the latest poetry version